lib/utils/isStandardSyntaxMathFunction.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Check whether a math function is standard
+ *
+ * @param {string} mathFunction
+ * @returns {boolean}
+ */
+module.exports = function isStandardSyntaxMathFunction(mathFunction) {
+	// SCSS variable
+	if (mathFunction.includes('$')) {
+		return false;
+	}
+
+	// Less variable
+	if (mathFunction.includes('@')) {
+		return false;
+	}
+
+	return true;
+};
```

Comment:
What about interpolation SCSS `#{var}`?

Reply:
Isn't it `#{$var}`?

Reply:
Oh, yes, what about `$` and `@` in css variable name, can these characters used?

Reply:
I couldn't get straight answer from the spec. But I tried in Chrome putting these symbols in different position in the custom property name, and all were invalid.

Reply:
Sounds good, approved